src/objects.rs

```diff
@@ -735,7 +735,7 @@ pub struct RawPathway {
     /// Location at which the pathway ends
     pub to_stop_id: String,
     /// Type of pathway between the specified (from_stop_id, to_stop_id) pair
-    pub mode: PathwayMode,
+    pub pathway_mode: PathwayMode,
```

Comment:
```suggestion
   #[serde(rename = "pathway_mode")]
    pub mode: PathwayMode,
```

Nice catch! Thank you.

We have a convention to not follow the bad design parts of GTFS where the type is repeated. A `Stop::stop_id` is in this crate a `Stop::id`.

With this suggestion, it will correctly read the gtfs file, while having a coherent naming